src/gluonnlp/layers.py

```diff
@@ -28,6 +28,8 @@
 import numpy as _np
 from typing import Union, Optional, List, Dict
 from .op import relative_position_bucket
+#from .attention_cell import MultiHeadAttentionCell
```

Comment:
This would be a circular import, as attention_cell also imports layers.

https://github.com/dmlc/gluon-nlp/blob/65c3047e479ca3927452b26536ca77943afe578d/src/gluonnlp/attention_cell.py#L25-L27

To solve this, two options are to either move SinusoidalPositionalEmbedding,\
                    BucketPositionalEmbedding,\
                    LearnedPositionalEmbedding out of the  layers.py into a new file and change the import in attention_cell. Or you can move AdapterModule into a new file. You can also come up with other solutions